ReactAndroid/src/main/java/com/facebook/react/common/TestIdUtil.java

```diff
@@ -0,0 +1,55 @@
+package com.facebook.react.common;
+
+import android.view.View;
+
+import com.facebook.react.common.annotations.VisibleForTesting;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class TestIdUtil {
+    private static final ConcurrentHashMap<String, Integer> mTestIds = new ConcurrentHashMap<>();
+    // Integer values in R.class are typically great in value.  Ids need to be greater than 0 to be
```

Comment:
nit: great in value -> large

Why do they need to be greater than 0 to be useful?

How about using 1 then (first integer after 0)?